scripts/seed.py

```diff
@@ -151,21 +200,25 @@ def create_challenge(title, start_date, end_date, host_team):
         terms_and_conditions=fake.paragraph(),
         submission_guidelines=fake.paragraph(),
         evaluation_details=fake.paragraph(),
-        evaluation_script=SimpleUploadedFile(evaluation_script.name, evaluation_script.read()),
+        evaluation_script=SimpleUploadedFile(
+            evaluation_script.name, evaluation_script.read()
+        ),
         approved_by_admin=True,
         leaderboard_description=fake.paragraph(),
         creator=host_team,
         published=True,
         enable_forum=True,
-        anonymous_leaderboard=False,
+        anonymous_leaderboard=anon_leaderboard,
         slug=slug,
```

Comment:
@MarchingVoxels This slug needs to be unique for every challenge. Please check the below error -
<img width="997" alt="Screenshot 2020-05-23 at 12 47 42 AM" src="https://user-images.githubusercontent.com/12206047/82722606-3ebfb500-9c96-11ea-89e1-cf4b01cf2d8c.png">